src/development/local.md

```diff
@@ -1,13 +1,14 @@
 # Set up your local development environment
 
-## Use the CLI
+While Platform.sh is great as a tool for hosting an application during both development and production, it's naturally not the ideal place to edit code.  You can't, in fact, as the file system is read-only (as it should be).  The proper place to edit your code is on your computer.
 
-Use the CLI to get a copy of your project, so that you can start working
-locally.
+You must have an [SSH key](/overview/tools.md#ssh) already configured on your account, and have both [Git](/overview/tools.md#git) and the [Platform.sh CLI](/overview/cli.md) installed before continuing.
 
-In order to copy your project locally, you'll need to know its project
-ID. The command `platform projects` will list all of the projects in
-your account.
+## Download the code
+
+If you don't already have a local Git clone of your code, you can easily download it.  From the Platform.sh UI copy the `platform get` command listed just under the action buttons on the right and run it locally on your system.
+
+Alternatively, you can use the CLI to get a copy of the code directly.  Run `platform projects` to list all of the projects in your account.
```

Comment:
IMO referring to the UI at this point adds a lot of extra thinking steps (1. understand what UI means, 2. find the UI URL for a particular project, 3. go to it, 4. maybe log in, 5. understand where the action buttons are, 6. copy and paste). Typing `platform get` is quicker... so I suggest:

> If you don't already have a local copy of your project's code, run `platform get` to download one. You can also run `platform projects` to list all of the projects in your account.

Reply:
I agree. I was confused when I read that sentence.